packages/jetbrains-plugin/src/main/resources/messages/TelemetryBundle.properties

```diff
@@ -0,0 +1,6 @@
+notification.group.name=MongoDB telemetry
+notification.title=MongoDB plugin telemetry
+notification.message=Anonymous telemetry is enabled by default because it helps us improve the plugin.
+action.disable-telemetry=Disable Telemetry
+settings.telemetry-collection-tooltip=Allow the collection of anonymous diagnostics and usage telemetry data to help improve the product.
+settings.telemetry-collection-checkbox=Enable telemetry
```

Comment:
Do we need to/can we include links to https://www.mongodb.com/legal/privacy/privacy-policy?

Reply:
That's a good point. I'm going to add a link on the notification and on the settings page to the privacy policy.